buildroot/package/dhcpcd/dhcpcd.mk

```diff
@@ -36,9 +36,11 @@ endef
 
 define DHCPCD_INSTALL_TARGET_CMDS
 	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) install DESTDIR=$(TARGET_DIR)
+	$(INSTALL) -D -m 0664 package/recovery/dhcpcd.conf $(TARGET_DIR)/etc/dhcpcd.conf
```

Comment:
Is this right? The other file in this PR seems to be in package/dhcpcd/... rather than package/recovery/... ?